tests/MSBuildDeviceIntegration/Tests/XASdkDeployTests.cs

```diff
@@ -83,6 +84,57 @@ public void DotNetInstallAndRun (bool isRelease, bool xamarinForms, bool? publis
 			Assert.IsTrue(didLaunch, "Activity should have started.");
 		}
 
+		[Test]
+		public void TypeAndMemberRemapping ([Values (false, true)] bool isRelease)
```

Comment:
This test never runs, for reasons I cannot currently explain or fathom.  It's in the downloaded `MSBuildDeviceIntegration.dll`.  It's not `[Values]`; `ApplicationRunsWithoutDebugger` *is* run.  It's not position in the file (moving it didn't change anything).

Meanwhile, I *can* run it locally, via the updated docs:

    ./dotnet-local.sh test bin/TestDebug/MSBuildDeviceIntegration/net6.0/MSBuildDeviceIntegration.dll --filter "Name~TypeAndMemberRemapping"

Fortunately, type and member remapping is *also* tested as part of `Java.Interop-Tests.dll`, included in `Mono.Android.NET-Tests.apk`, so I'm confident it works.  It would just be nice if this test *also* worked.  Or at least *ran*.